README.md

```diff
@@ -17,58 +17,65 @@ Originally faces.js was made for [Basketball GM](https://basketball-gm.com/) and
 ## Use
 
 Import it with ES modules:
-
+```javascript
     import { display, generate } from "facesjs";
+```
```

Comment:
Can you make the spacing consistent, like always have a line break before and after the code blocks?

Reply:
Sure! I'll make the changes now, please let me know if that is what you meant 🙂